packages/web-components/fast-foundation/src/dialog/README.md

```diff
@@ -5,7 +5,25 @@ sidebar_label: dialog
 custom_edit_url: https://github.com/microsoft/fast/edit/master/packages/web-components/fast-foundation/src/dialog/README.md
 ---
 
-A web component implementation of a [dialog](https://w3c.github.io/aria-practices/#dialog_modal) which can either be modal or non-modal.
+As defined by the [W3C](https://w3c.github.io/aria-practices/#dialog_modal):
+
+> A dialog is a window overlaid on either the primary window or another dialog window. Windows under a modal dialog are inert. That is, users cannot interact with content outside an active dialog window. Inert content outside an active dialog is typically visually obscured or dimmed so it is difficult to discern, and in some implementations, attempts to interact with the inert content cause the dialog to close.
+>
+> Like non-modal dialogs, modal dialogs contain their tab sequence. That is, Tab and Shift + Tab do not move focus outside the dialog. However, unlike most non-modal dialogs, modal dialogs do not provide means for moving keyboard focus outside the dialog window without closing the dialog.
```

Comment:
One of the reasons that modal/non-modal was initially included was because the W3C definition trends strongly towards "modal" which without a careful reading could create some incorrect assumptions. I'm good with this change, but something to be aware of.